integration/eth2network/eth2_network.go

```diff
@@ -303,10 +306,8 @@ func (n *Impl) Start() error {
 		}()
 	}
 
-	// this locks the process waiting up to one minute for the event to happen
-	timeoutCtx, cancelCtx := context.WithTimeout(context.Background(), 1*time.Minute)
-	defer cancelCtx()
-	return n.waitForMergeEvent(timeoutCtx, startTime)
+	// blocking wait until the network reaches the Merge
+	return n.waitForMergeEvent(startTime)
```

Comment:
It makes the error handling for the timeout easier. It's actively testing for a timeout instead of returning a context deadline that can be from either the rpc call or the code thats been running for too long.